test/ClangImporter/pch-bridging-header.swift

```diff
@@ -46,6 +46,12 @@
 // RUN: not %target-swift-frontend -typecheck %s -import-objc-header %S/Inputs/sdk-bridging-header.h -pch-output-dir %t/no-pch -pch-disable-validation 2>&1 | %FileCheck %s -check-prefix=NO-VALIDATION
 // NO-VALIDATION: PCH file {{.*}} not found
 
+// Test that -Xcc options are considered in the PCH hash.
+// RUN: %target-swift-frontend -emit-pch -pch-output-dir %t/pch-Xcc %S/Inputs/sdk-bridging-header.h
+// RUN: %target-swift-frontend -emit-pch -pch-output-dir %t/pch-Xcc %S/Inputs/sdk-bridging-header.h -Xcc -Ifoo
+// RUN: %target-swift-frontend -emit-pch -pch-output-dir %t/pch-Xcc %S/Inputs/sdk-bridging-header.h -Xcc -Ibar
+// RUN: ls %t/pch-Xcc/*swift*clang*.pch | count 3
```

Comment:
I think `count` just prints the number of elements, it doesn't validate how many elements were passed

Reply:
It does, actually! I made sure to test this with the 5.1 compiler.

(I think it's an LLVM tool, not a shell builtin.)

Reply:
https://github.com/apple/swift-llvm/blob/5bccd3526fd93ceb9e9de4ece0ffa2c5e26a9a02/utils/count/count.c#L43